iam.tf

```diff
@@ -48,3 +48,16 @@ resource "google_service_account_iam_member" "tenant_orchestration_impersonate_t
   role               = "roles/iam.serviceAccountTokenCreator"
   member             = "serviceAccount:${var.tenant_orchestration_sa}"
 }
+
+##########################################
+## Grant default cloud build account Storage Object Viewer role
+##########################################
+data "google_project" "data_plane_project" {
```

Comment:
can you move this to the [private.metrics.tf](https://github.com/LiveRamp/terraform-google-identity-engine-data-plane/blob/main/private.metrics.tf) file. 

Lets keep all the metrics resources in one place